lib/fileset/README.md

```diff
@@ -41,9 +41,16 @@ An attribute set with these values:
 - `_type` (constant string `"fileset"`):
   Tag to indicate this value is a file set.
 
-- `_internalVersion` (constant `2`, the current version):
+- `_internalVersion` (constant `3`, the current version):
```

Comment:
Should this be in a file like `fileset/internal/README.md`?

Reply:
I view `lib/fileset/README.md` as the internal contributor documentation, where I think this makes sense to have. It's not part of the public interface, but you need to know about it to contribute, which can mean to change the internals

Reply:
It should link to the user docs.